helmfile.d/values/gatekeeper/gatekeeper.yaml.gotmpl

```diff
@@ -6,6 +6,8 @@ validatingWebhookExemptNamespacesLabels:
   - kube-public
   - kube-node-lease
   - calico-system
+  - fluentd
+  - fluentd-system
```

Comment:
Why are these here?

Reply:
We are configuring the pod security policies of fluentd, so if it needs more permission, update the policies.